test/bin/start_osbuild_workers.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# This script should be run on the image building host to manage the
+# number of workers for creating images in parallel.
+
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPTDIR}/common.sh"
+
+if [ $# -eq 0 ]; then
+    error "Usage: ${0} number-of-workers"
+    exit 1
+fi
+num_workers="${1}"
+
+# Loop from 2 because we should already have at least 1 worker.
+for i in $(seq 2 "${num_workers}"); do
+    sudo systemctl start "osbuild-worker@${i}.service"
```

Comment:
What stops the workers after they are finished?

Reply:
The command does not seem to report an error, no.

Nothing stops the workers. In CI, we're going to kill the entire host. On a local system, I expect the user to manage the processes and this is just a convenience script for starting several at once.

Reply:
Since we have the "cancel" procedure, it seems like the right place to spin down the workers.
This is mostly required for the devenv, of course. But, also for consistency.